File: www/item-series/demo-grid-light.m
#import <UIKit/UIKit.h>
#import <Highcharts/Highcharts.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    HIChartView *chartView = [[HIChartView alloc] initWithFrame:self.view.bounds];

    chartView.theme = @"grid-light";
    
    HIOptions *options = [[HIOptions alloc] init];
    
    HIChart *chart = [[HIChart alloc] init];
    chart.type = @"item";

    HITitle *title = [[HITitle alloc] init];
    title.text = @"Highcharts item chart";
    
    HIXAxis *xAxis = [[HIXAxis alloc] init];
    xAxis.categories = @[@"Jan", @"Feb", @"Mar", @"Apr", @"May", @"Jun", @"Jul", @"Aug", @"Sep", @"Oct", @"Nov", @"Dec"];
    
    HIYAxis *yAxis = [[HIYAxis alloc] init];
    yAxis.gridLineWidth = @0;
    yAxis.labels = [[HILabels alloc] init];
    yAxis.labels.enabled = [[NSNumber alloc] initWithBool:false];
    yAxis.title = [[HITitle alloc] init];
    
    HIPlotOptions *plotOptions = [[HIPlotOptions alloc] init];
    plotOptions.series = [[HISeries alloc] init];
    plotOptions.series.stacking = @"normal";
    
    HISeries *series1 = [[HISeries alloc] init];
    series1.name = @"Items bought";
    series1.data = [NSMutableArray arrayWithObjects:@5, @3, @4, nil];
    series1.color = [[HIColor alloc] initWithName:@"green"];
    
    HISeries *series2 = [[HISeries alloc] init];
    series2.name = @"Items sold";
    series2.data = [NSMutableArray arrayWithObjects:@0, @2, @1, nil];
    series2.color = [[HIColor alloc] initWithName:@"red"];
    options.series = [NSMutableArray arrayWithObjects:series1, series2, nil];
    
    options.chart = chart;
    options.title = title;
    options.xAxis = [NSMutableArray arrayWithObjects:xAxis, nil];
    options.yAxis = [NSMutableArray arrayWithObjects:yAxis, nil];
    options.plotOptions = plotOptions;
    
    chartView.options = options;
    
    [self.view addSubview:chartView];
}

@end

File: www-swift/line-labels/demo-grid-light.swift
import Highcharts
import UIKit

class ViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()

    let chartView = HIChartView(frame: view.bounds)
    chartView.theme = "grid-light"
    chartView.plugins = ["series-label"]

    let options = HIOptions()

    let chart = HIChart()
    chart.type = "line"
    options.chart = chart

    let title = HITitle()
    title.text = "Monthly Average Temperature"
    options.title = title

    let subtitle = HISubtitle()
    title.text = "Source: WorldClimate.com"
    options.subtitle = subtitle

    let xAxis = HIXAxis()
    xAxis.categories = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
    options.xAxis = [xAxis]

    let yAxis = HIYAxis()
    yAxis.title = HITitle()
    yAxis.title.text = "Temperature (°C)"
    options.yAxis = [yAxis]

    let plotOptions = HIPlotOptions()
    plotOptions.line = HILine()

    let dataLabels = HIDataLabels()
    dataLabels.enabled = true
    plotOptions.line.dataLabels = [dataLabels]

    plotOptions.line.enableMouseTracking = false
    options.plotOptions = plotOptions

    let tokyo = HILine()
    tokyo.name = "Tokyo"
    tokyo.data = [7.0, 6.9, 9.5, 14.5, 18.4, 21.5, 25.2, 26.5, 23.3, 18.3, 13.9, 9.6]

    let london = HILine()
    london.name = "London"
    london.data = [3.9, 4.2, 5.7, 8.5, 11.9, 15.2, 17.0, 16.6, 14.2, 10.3, 6.6, 4.8]

    options.series = [tokyo, london]

    chartView.options = options

    self.view.addSubview(chartView)
  }

}
